devices/pine64-pinephone/kernel/config.aarch64

```diff
@@ -109,7 +109,8 @@ CONFIG_TASKSTATS=y
 CONFIG_TASK_DELAY_ACCT=y
 CONFIG_TASK_XACCT=y
 CONFIG_TASK_IO_ACCOUNTING=y
-# CONFIG_PSI is not set
+CONFIG_PSI=y
+CONFIG_PSI_DEFAULT_DISABLED=y
```

Comment:
Are there reasons we would keep it disabled by default?

Reply:
I thought it is closer to how it was before. I am fine with enabling it by default too. Guess there is no reason to make it to complex. Initially my plan was to combine it with something like NixOS/nixpkgs#147365.

Reply:
https://github.com/torvalds/linux/blob/c5c17547b778975b3d83a73c8d84e8fb5ecf3ba5/init/Kconfig#L646

Let's use the default.

```suggestion
CONFIG_PSI=y
# CONFIG_PSI_DEFAULT_DISABLED is not set
```

The option description states the overhead is too low to affect "real world" tasks.